Assets/Shaders/SnapshotCameraFlash.shader

```diff
@@ -73,6 +73,7 @@ Shader "Unlit/Snapshot Camera Flash"
                 col.a *= _Alpha;
                 // apply fog
                 UNITY_APPLY_FOG(i.fogCoord, col);
+                col.a = 1.0;
```

Comment:
Can you remove line 73?

Reply:
Yes, also I'd like to consider this a temporary fix until we can look at the shader (and other screenshot shaders) properly :D

Reply:
makes sense. Need any testing? Just approval?

Reply:
Would be great if someone can confirm this hotfix works, I'm not sure if/how config works on quest either, I've only tested against desktop.